### PR TITLE
RFC Improvements and Package Version Updates

### PR DESCRIPTION
**Pull request by VRamakrishna**
Updated the Fabric Interoperation Chaincode `libs/utils` version.
Fixed permissioning issues in containerized Fabric Driver and Fabric IIN Agent.
Upgraded Fabric testnets to latest Fabric and Fabric-CA versions.
Updated Data Sharing RFCs, partially fixing #238 .

### CODE REVIEW

File: samples/fabric/simplestatewithacl/go.mod [hunk 1]
```diff
@@ -5,7 +5,7 @@ go 1.15
 require (
 	github.com/golang/protobuf v1.5.2
 	github.com/hyperledger-labs/weaver-dlt-interoperability/core/network/fabric-interop-cc/libs/testutils v0.0.0-20210909191523-de832057a3ab
-	github.com/hyperledger-labs/weaver-dlt-interoperability/core/network/fabric-interop-cc/libs/utils v1.3.1
+	github.com/hyperledger-labs/weaver-dlt-interoperability/core/network/fabric-interop-cc/libs/utils v1.5.6
```
Comment (sanvenDev) [inline samples/fabric/simplestatewithacl/go.mod:8]: won't you also need to update go.sum as well?
there's a tool go-checksum that can be used, or also you can do that in separate PR after modules are published.

Comment (VRamakrishna) [reply samples/fabric/simplestatewithacl/go.mod:8]: But the `go.sum` won't even be generated unless the `1.5.6` version of `libs/utils` gets published, no?

File: rfcs/formats/views/definition.md [hunk 2]
```diff
@@ -3,80 +3,91 @@
...
+```
+
+## Commitment (Future Work)
+
+```protobuf
+message Commitment {
+    // How is the commitment exchanged/located?
+    string address = 3;
+	// Cryptographic commitment scheme
```
Comment (sanvenDev) [inline rfcs/formats/views/definition.md:87]: indentation issue

File: rfcs/protocols/data-sharing/fabric.md [hunk 3]
```diff
@@ -8,7 +8,50 @@
 - RFC: 02-002
 - Authors: Venkatraman Ramakrishna, Sandeep Nishad, Krishnasuri Narayanam, Dhinakaran Vinayagamurthy
 - Status: Proposed
-- Since: 01-Dec-2020
+- Since: 02-Jan-2023
 
 ## Summary
 
+This document specifies the Hyperledger Fabric implementation of modules, and application adaptation guidelines, for the asset transfer protocol.
```
Comment (sanvenDev) [inline rfcs/protocols/data-sharing/fabric.md:15]: Why does data-sharing rfc says this is for asset transfer? Even the next line is also about asset transfer

File: rfcs/protocols/data-sharing/fabric.md [hunk 4]
```diff
@@ -8,7 +8,50 @@
...
+Within Weaver, the protocol units to operate on the asset being transferred will be implemented in the Fabric Interoperation Chaincode, which is the [interoperation module](../../models/infrastructure/interoperation-modules.md) for a Fabric-based network.
+- The functions are meant to be called only by relay drivers with special wallet identities that are recognized by the Fabric Interoperation Chaincode at bootstrap.
```
Comment (sanvenDev) [inline rfcs/protocols/data-sharing/fabric.md:17]: If its just one point in bullet, would a new para be better?

File: samples/fabric/simplestatewithacl/go.mod (hunk 1 above)
Comment (sanvenDev) [reply samples/fabric/simplestatewithacl/go.mod:8]: yeah it won't be generated automatically, you can do it in next PR then, when you enable the workflows

**Review by sanvenDev — APPROVED** [review]

File: rfcs/protocols/data-sharing/fabric.md (hunk 3 above)
Comment (VRamakrishna) [reply rfcs/protocols/data-sharing/fabric.md:15]: Typo. This file was initially copied from the corresponding asset transfer RFC.

File: rfcs/protocols/data-sharing/fabric.md (hunk 4 above)
Comment (VRamakrishna) [reply rfcs/protocols/data-sharing/fabric.md:17]: I made the entire section a bullet list. Also made a similar change in the Fabric asset transfer RFC.
